qcodes/instrument_drivers/Keysight/keysightb1500/constants.py

```diff
@@ -618,6 +618,42 @@ class MeasurementMode(IntEnum):
         Ls_D = 401
         Ls_Q = 402
 
+    class Name(StrEnum):
+        R = 'Resistance'
+        G = 'Conductance'
+        Z = 'Impedance'
+        Y = 'Admittance'
+        Cp = 'Parallel Capacitance'
+        Cs = 'Series Capacitance'
+        Lp = 'Parallel Inductance'
+        Ls = 'Series Inductance'
+        X = 'Reactance'
+        B = 'Susceptance'
+        THETA_RAD = 'Phase'
+        THETA_DEG = 'Phase'
+        D = 'Dissipation Factor'
+        Q = 'Quality Factor'
+        Rp = 'Parallel Resistance'
+        Rs = 'Series Resistance'
+
+    class Unit(StrEnum):
+        R = 'ohm'
+        G = 'S'
+        Z = 'ohm'
+        Y = 'S'
+        Cp = 'F'
+        Cs = 'F'
+        Lp = 'H'
+        Ls = 'H'
+        X = 'ohm'
+        B = 'S'
+        THETA_RAD = 'radian'
+        THETA_DEG = 'degree'
+        D = 'unit'
+        Q = 'unit'
```

Comment:
unit-less parameters better have empty string as a unit
```suggestion
        D = ''
        Q = ''
```

Reply:
That leaves things ambiguous. Was unit known and not set or was it really unitless? I personally prefer having 'unit' if things are unitless and 'arb. unit' if things are an arbitrary unit. If you strongly feel otherwise, I can change it to ''

Reply:
> That leaves things ambiguous

agree. However, using `''` for unit-less parameter is consistent with other drivers and the parameter's docstring itself:

```
#: The unit of measure. Use ``''`` for unitless.
self.unit = unit if unit is not None else ''
```

So i believe we have to use `''` to be consistent.